### PR TITLE
Internal: AI builder info message [ED-11529]

### PR DESCRIPTION
**Pull request by rotemee**

### CODE REVIEW

File: modules/ai/assets/js/editor/hooks/use-introduction.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+
+export default function useIntroduction( key ) {
+	const [ isViewed, setIsViewed ] = useState( !! window.ElementorConfig?.user?.introduction?.[ key ] );
```
Comment (StyleShit) [inline modules/ai/assets/js/editor/hooks/use-introduction.js:4]:
```suggestion
	const [ isViewed, setIsViewed ] = useState( !! window.elementor.config?.user?.introduction?.[ key ] );
```

File: modules/ai/assets/js/editor/hooks/use-introduction.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+			elementorCommon.ajax.addRequest( 'introduction_viewed', {
+				data: { introductionKey: key },
+				error: () => reject(),
+				success: () => {
+					setIsViewed( true );
+
+					if ( window.ElementorConfig?.user?.introduction ) {
+						window.ElementorConfig.user.introduction[ key ] = true;
+					}
```
Comment (StyleShit) [inline modules/ai/assets/js/editor/hooks/use-introduction.js:24]: same as above

File: modules/ai/assets/js/editor/layout-content.js [hunk 3]
```diff
@@ -6,9 +7,13 @@ import UpgradeChip from './components/upgrade-chip';
 import useUserInfo from './hooks/use-user-info';
 import WizardDialog from './components/wizard-dialog';
 import LayoutDialog from './pages/form-layout/components/layout-dialog';
+import { Alert } from '@elementor/ui';
+import useIntroduction from './hooks/use-introduction';
 
 const LayoutContent = ( { onClose, onConnect, onGenerationStart, onGenerationEnd, onInsert, onSelect } ) => {
 	const { isLoading, isConnected, isGetStarted, connectUrl, fetchData, hasSubscription, credits, usagePercentage } = useUserInfo();
+	const { isViewed, markAsViewed } = useIntroduction( 'e-ai-builder-no-images-info' );
+	const [ showInfoAlert, setShowInfoAlert ] = useState( () => ! isViewed );
```
Comment (StyleShit) [inline modules/ai/assets/js/editor/layout-content.js:16]: pretty sure you can use only the introduction state

**Review by StyleShit — CHANGES_REQUESTED** [review]

**Review by StyleShit — APPROVED** [review]

File: modules/ai/assets/js/editor/hooks/use-introduction.js (hunk 1 above)
Comment (rotemee) [reply modules/ai/assets/js/editor/hooks/use-introduction.js]: Done.

File: modules/ai/assets/js/editor/hooks/use-introduction.js (hunk 2 above)
Comment (rotemee) [reply modules/ai/assets/js/editor/hooks/use-introduction.js]: Done.

File: modules/ai/assets/js/editor/layout-content.js (hunk 3 above)
Comment (rotemee) [reply modules/ai/assets/js/editor/layout-content.js]: Done.

**Pull request merged**